README.md

```diff
@@ -1,11 +1,16 @@
 # Versity
-Versity App will be a companion app for university students which will help them with academics and socials, the two aspects of university that our team believes an ‘all-rounder’ student should excel in.
+## 👨🏻‍💻 About the project
```

Comment:
There need not be any special characters before 'About the project'. A simple heading is fine